crates/sui-graphql-e2e-tests/tests/available_range/available_range.exp

```diff
@@ -35,20 +35,20 @@ Response: {
   "data": {
     "availableRange": {
       "first": {
-        "digest": "9hCFi7xotaerYwZbFJUdHPNijJtUKMBpxkukoyp9FcTA",
+        "digest": "8Bt5oYg9P2iZPXyv44guxAGhXNwd8Pi93EZ8ye9SzcmU",
         "sequenceNumber": 0
       },
       "last": {
-        "digest": "4KMiBygiY5pNj53QJKDmHGkcXywEUrEwg85KRQFjAhty",
+        "digest": "ASDG27SZyAj6xVUqnA8DQSk5ofTKaxgddnRSMJG8kiCT",
         "sequenceNumber": 2
       }
     },
     "first": {
-      "digest": "9hCFi7xotaerYwZbFJUdHPNijJtUKMBpxkukoyp9FcTA",
+      "digest": "8Bt5oYg9P2iZPXyv44guxAGhXNwd8Pi93EZ8ye9SzcmU",
       "sequenceNumber": 0
     },
     "last": {
-      "digest": "4KMiBygiY5pNj53QJKDmHGkcXywEUrEwg85KRQFjAhty",
```

Comment:
curious why all these values changed?

Reply:
They just generally have shifted every time there is a new protocol version. So my understanding is there's some deterministic state (e.g., genesis) that changes with the new protocol version (similar to the configs that also need to be updated with new values after a protocol version bump).